shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -126,4 +126,20 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
   [RCTPushNotificationManager didReceiveLocalNotification:notification];
 }
 
+- (void)applicationDidEnterBackground:(UIApplication *)application
```

Comment:
I think you might want to use `applicationWillResignActive`.
https://stackoverflow.com/questions/18959411/controlling-the-screenshot-in-the-ios-7-multitasking-switcher